.github/Makefile

```diff
@@ -1,3 +1,5 @@
+.SILENT:
+MAKEFLAGS += --no-print-directory
```

Comment:
This helps when running `$(MAKE)` in any make goal since it removes the verbosity in the log output regarding switching from one dir to another...